quickstart-sda.md

```diff
@@ -0,0 +1,239 @@
+# Quickstart guide to the SDA
+
+This document contains information on how to work with the Sensitive Data Archive (SDA) from the GDI user perspective.
+
+## Brief description of the storage and interfaces stack
+
+The storage and interfaces software stack for the GDI-starter-kit consists of the following services:
+
+| Component     | Description |
+|---------------|------|
+| broker        | RabbitMQ based message broker, [SDA-MQ](https://github.com/neicnordic/sda-mq). |
+| database      | Postgres database, [SDA-DB](https://github.com/neicnordic/sda-db). |
+| storage       | S3 object store, demo uses Minio S3 |
+| mock-oidc     | A python implementation of a mock-oidc inplace of LS-AAI |
+| s3inbox       | Proxy inbox to the S3 backend store, [SDA-S3Proxy](https://github.com/neicnordic/sda-s3proxy) |
+| download      | Data out solution for downloading files from the SDA, [SDA-download](https://github.com/neicnordic/sda-download) |
+| SDA-pipeline     | The ingestion pipeline of the SDA, [SDA-pipeline](https://github.com/neicnordic/sda-pipeline). This comprises of the following core components: `ingest`, `verify`, `finalize` and `mapper`.|
+
+Detailed documentation on the `sda-pipeline` can be found at: https://neicnordic.github.io/sda-pipeline/pkg/sda-pipeline/.
+
+NeIC Sensitive Data Archive documentation can be found at: https://neic-sda.readthedocs.io/en/latest/ .
+
+## Deployment
+
+The storage and interfaces stack can be deployed with the use of the provided `docker-compose.yml` file by running
+
+```cmd
+docker compose up -d
+```
+
+from the root of this repo. Configuration can be customized by changing the `config/config.yml` file.
+
+### Adding TLS to internet facing services
+
+Internet facing services such as e.g. `s3inbox`, `download` and `mock-oidc`, need to be secured via TLS certification. This can be most conveniently achieved by using [letsencrypt](https://letsencrypt.org/getting-started/) as Certificate Authority. Assuming shell access to your web host, a convenient way to set this up is through  installing certbot (or any other ACME client supported by letsencrypt). Detailed instructions on setting up `certbot` for different system configurations can be found [here](https://certbot.eff.org/).
```

Comment:
Both `certbot` and  `letsencrypt` should be written like this, in monotype, if they refer to the commands, or as "Certbot" and "Let's Encrypt" if you use them as names (of the two projects).

Reply:
Good point, done.